modules/openy_features/openy_taxonomy/modules/openy_txnm_blog_category/openy_txnm_blog_category.install

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Update routines.
+ */
+
+/**
+ * Add Taxonomy content field.
+ */
+function openy_txnm_blog_category_update_8001() {
+  $config_dir = drupal_get_path('module', 'openy_txnm_blog_category') . '/config/install';
+  $config_importer = \Drupal::service('openy_upgrade_tool.importer');
+  $config_importer->setDirectory($config_dir);
+  $config_importer->importConfigs([
+    'core.entity_form_display.taxonomy_term.blog_category.default',
+    'core.entity_view_display.taxonomy_term.blog_category.default',
+    'field.field.taxonomy_term.blog_category.field_taxonomy_content',
+    'field.storage.taxonomy_term.field_taxonomy_content'
```

Comment:
I think storage sjould live in `openy_taxonomy` module, because it can be reused by other taxonomy pages.